README.md

```diff
@@ -37,7 +37,7 @@ Join `#buildkit` channel on [Docker Community Slack](https://dockr.ly/comm-slack
 
 > **Note**
 >
-> [BuildKit has been integrated to `docker build` since Docker 18.09](https://docs.docker.com/develop/develop-images/build_enhancements/).
+> [BuildKit is used by the default implementation of `docker build` since Docker Engine 23.0]([https://docs.docker.com/develop/develop-images/build_enhancements/](https://docs.docker.com/build/architecture/)).
```

Comment:
Link looks malformed

```suggestion
> [BuildKit is used by the default implementation of `docker build` since Docker Engine 23.0](https://docs.docker.com/build/architecture/).
```

Reply:
Whoops!